variables.tf

```diff
@@ -199,7 +199,7 @@ variable "restore_to_point_in_time" {
   type = list(object({
     source_cluster_identifier  = string
     restore_type               = optional(string, "copy-on-write")
-    use_latest_restorable_time = optional(bool, true)
+    use_latest_restorable_time = optional(bool, null)
```

Comment:
It's important that we don't change the default behavior and create a breaking change

Reply:
Agreed! [These lines](https://github.com/cloudposse/terraform-aws-rds-cluster/pull/217/files#diff-dc46acf24afd63ef8c556b77c126ccc6e578bc87e3aa09a931f33d9bf2532fbbR16-R20) preserve the existing behavior. Is there a better way to accomplish this (e.g. adding a new variable and deprecating this one)?

Reply:
I see now how you did that with the local value. I think this is inherently confusing logic with the redirection from the default value. 

How about instead if `restore_to_time` is defined, then ignore the value given by `use_latest_restorable_time` and make set the input in the dynamic block to `null`. Otherwise, use the value defined in `use_latest_restorable_time`

Reply:
```hcl

  dynamic "restore_to_point_in_time" {
    for_each = var.restore_to_point_in_time
    content {
      source_cluster_identifier  = restore_to_point_in_time.value.source_cluster_identifier
      restore_type               = restore_to_point_in_time.value.restore_type
      # use_latest_restorable_time and restore_to_time are mutually exclusive.
      # If restore_to_time is given, then we ignore use_latest_restorable_time
      use_latest_restorable_time = restore_to_point_in_time.value.restore_to_time != null ? null : restore_to_point_in_time.value.use_latest_restorable_time
      restore_to_time            = restore_to_point_in_time.value.restore_to_time
    }
  }
  
  ```

Reply:
great suggestion, that's much easier to understand! I was definitely scratching my head trying to come up with the local value 😄 

tested this out locally and all the permutations of these parameters work as expected